.github/workflows/CI.yml

```diff
@@ -25,7 +28,12 @@ jobs:
           npx commitlint --version
       - name: Install yarn
         run: |
-          sudo npm install -g yarn
+          npm install -g yarn
+          yarn add --dev jest typescript ts-jest @types/jest
+      - name: Install commitlint
+        run: |
+          npm install conventional-changelog-conventionalcommits
+          npm install commitlint@17.4.2
```

Comment:
@realmarv isn't this already done in commitlint.sh? so we don't need it here

Reply:
We don't run commitlint.sh in tests job